src/App.js

```diff
@@ -49,8 +53,9 @@ class App extends Component {
                height={item.height}     // Height - used for a CSS style to position on the screen
                layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
                key={item.id}            // Key - to help React with performance
-
+                type={item.type}
                // Additional props (event callbacks, etc.) can be passed here
+                score={this.onItemScoreClicked }
```

Comment:
Calling this `score` is not helpful to the reader. Is it a variable? a Function? Best to call out that it is a callback in the prop name.